src/CMakeLists.txt

```diff
@@ -173,6 +173,11 @@ celeritas_add_library(celeritas ${SOURCES})
 celeritas_strip_alias(celeritas_target celeritas)
 add_library(Celeritas::Core ALIAS ${celeritas_target})
 
+find_package(OpenMP)
+if(OPENMP_FOUND)
+  celeritas_target_link_libraries(celeritas PRIVATE OpenMP::OpenMP_CXX)
+endif()
```

Comment:
We're probably going to need an `else()` to add `-Wno-unknown-pragmas`...

Reply:
It looks like we'd need to add a `celeritas_target_compile_options` to pass flags to the appropriate libraries. Should I go ahead and implement that or did you have something else in mind?